### ISSUE TITLE
Control the normalized value of scalar pixel types

### ISSUE THREAD

**Issue by andy-sweet** [chanzuckerberg/idetik#32]
In #28, we added support for loading uint16 image data from OME-Zarr sources. The fragment shader performs normalization of the uint16 texture values by converting to float, then dividing by 256.0 which was chosen to present a specific example dataset well.

Instead, we should parametrize this normalization and allow these values to be passed through as part of the definition of the `ImageLayer`. This gives clients of the library (e.g. the prototype application) a way to control the presentation of scalar values.

**Comment by andy-sweet** [comment]
For multi-channel images, I believe we can define [an array of `vec2` in the fragment uniform](https://www.khronos.org/opengl/wiki/Data_Type_(GLSL)#Arrays_of_arrays) and pass the flattened version of that array from TypeScript.

**Comment by aganders3** [comment]
I can't tell from the documentation here, can this be dynamically sized?

If not I have some ideas:
* set the size of the uniform in a pre-compilation step
* use a fixed upper-bound size (64?) and a separate uniform to indicate how may channels are used (may need this anyway, or get it from the length of the texture array holding the channel images)
* use an Nx2 texture

**Comment by andy-sweet** [comment]
> I can't tell from the documentation here, can this be dynamically sized?

I don't think so. I also remember reading there is a hard limit (something like 1024).

Your suggestions sound reasonable, with the second option sounding like the easiest and maybe the one I'd pick first.
